salt/auth/__init__.py

```diff
@@ -680,11 +680,6 @@ def _send_token_request(self, load):
                                                                 master_uri=master_uri)
```

Comment:
Yeah, it is. I was thinking about leaving those there in case we ever implemented some other transport options, but I can remove that for now and the others you mentioned below.

Reply:
I thought about it too. But then thought that not having that `if` will follow the developer of the new transport to a better path that is followed by both `tcp` and `zeromq` transports that don't need that `if` case. :smile: